crates/astria-sequencer/src/sequence/action.rs

```diff
@@ -88,6 +90,25 @@ impl ActionHandler for Action {
         Ok(())
     }
 
+    fn check_stateless(&self) -> Result<()> {
+        ensure!(
+            !self.chain_id.is_empty(),
+            "cannot have empty chain ID for sequence action",
+        );
+        ensure!(
+            self.chain_id.len() <= MAX_CHAIN_ID_LENGTH,
+            "chain ID cannot be longer than {} bytes",
+            MAX_CHAIN_ID_LENGTH,
+        );
+
+        // TODO: do we want to place a maximum on the size of the data?
```

Comment:
I think this is fine to leave as open TODO, but lets make issue and tag with number here

Reply:
done